xradar/model.py

```diff
@@ -300,6 +316,8 @@
         ("dynamic_range_db_hx", None),
         ("dynamic_range_db_vx", None),
         ("dbz_correction", None),
+        ("half_power_beam_wigth_h", None),
```

Comment:
Sorry, this somehow caught my eye :) Is this a typo or intentional? "wigth" or "width"